common/src/main/java/org/dash/wallet/common/ui/enter_amount/AmountView.kt

```diff
@@ -207,17 +236,11 @@ class AmountView(context: Context, attrs: AttributeSet) : ConstraintLayout(conte
         binding.resultCurrencyToggle.isVisible = showCurrencySelector && dashToFiat
 
         if (dashToFiat) {
-            binding.inputSymbolDash.isVisible = isCurrencySymbolFirst
-            binding.inputSymbolDashPostfix.isVisible = !isCurrencySymbolFirst
-
```

Comment:
The symbol is now inserted into the Text Span, rather than being in a separate view.